src/Turbo/Twig/StreamExtension.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Kévin Dunglas <kevin@dunglas.fr>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Turbo\Twig;
+
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+/**
+ * Twig helpers to generated the Turbo Stream elements.
+ *
+ * @author Kévin Dunglas <kevin@dunglas.fr>
+ *
+ * @experimental
+ */
+final class StreamExtension extends AbstractExtension
+{
+    private const ACTIONS = [
+        'append' => true,
+        'prepend' => true,
+        'replace' => true,
+        'update' => true,
+        'remove' => true,
+    ];
+
+    private $mercureHub;
+
+    public function __construct(?string $mercureHub = null)
+    {
+        $this->mercureHub = $mercureHub;
+    }
+
+    public function getFunctions(): iterable
+    {
+        yield new TwigFunction('turbo_stream_start', [$this, 'turboStreamStart'], ['is_safe' => ['html']]);
+        yield new TwigFunction('turbo_stream_end', [$this, 'turboStreamEnd'], ['is_safe' => ['html']]);
+        yield new TwigFunction('turbo_stream_from_start', [$this, 'turboStreamFromStart'], ['is_safe' => ['html']]);
```

Comment:
In my PR, I chose not to wrap content inside this div, because I think it creates the false impression that only the content inside the div will be affected by the stream, which isn't the case.

That's why I declare a single `turbo_stream_from` function that creates an empty div: https://github.com/tgalopin/ux/blob/31e064fb2ab6801fd6bd62b9a1505c6125fb67da/src/Turbo/README.md#3-create-real-time-applications-with-turbo-streams

WDYT?